Xamarin.Forms.Core/WebView.cs

```diff
@@ -145,5 +177,39 @@ public void SendNavigating(WebNavigatingEventArgs args)
 		{
 			return _platformConfigurationRegistry.Value.On<T>();
 		}
+
+		static string EscapeJsString(string js)
+		{
+			if (!js.Contains("'"))
```

Comment:
check if this is null or it will throw NRE

Reply:
Good catch, will do